content/engineering/cloud/saas/index.md

```diff
@@ -1,41 +1,207 @@
+- [Quinn Keast](../../../company/team/index.md#quinn-keast) (he/him) - [Product Designer](../../../product/roles/index.md#product-designer)
+- [Rafal Leszczynski](../../../company/team/index.md#rafal-leszczynski) (he/him) - [Engineering Manager](../../roles.md#engineering-manager)
+  - [Joe Chen](../../../company/team/index.md#joe-chen) (he/him)
+  - [Artem Ruts](../../../company/team/index.md#artem-ruts) (he/him)
+  - [Milan Freml](../../../company/team/index.md#milan-freml) (he/him)
+  - [Rafal Gajdulewicz](../../../company/team/index.md#rafal-gajdulewicz) (he/him)
+  - P. R. (he/him) - starting 2022.01
+
+We’re hiring! [Check out our open roles](https://boards.greenhouse.io/sourcegraph91/jobs/4101082004).
+
+## Partner Teams
+
+The Cloud SaaS team works alongside several other teams within the Cloud organization at Sourcegraph. You can find more information about their teams and goals on the respective pages:
+
+- [Cloud Growth](../growth)
+- [Cloud DevOps](../devops)
+- [Extensibility](../extensibility)
+- [Security](../security)
+
+## News and updates
+
+- [2021.10.13 monthly update](https://groups.google.com/a/sourcegraph.com/g/engineering-team-status/c/3p0Pj2-RfVY?hl=en)
+
+## How we work
+
+### How to contact the team and ask for help
 
 - For cloud users with urgent help requests reach out to our support team at [support@sourcegraph.com](mailto:support@sourcegraph.com).
-- For internal Sourcegraph teammates, join us in #cloud-saas to ask questions or request help from our team.
-- For cloud users with feature requests, please reach out to our product manager, Ryan, at [ryphil@sourcegraph.com](mailto:ryphil@sourcegraph.com) and include "Cloud Feature Request:" in your subject line.
+- For emergencies and incidents, alert the team using Slack command `/genie alert [message] for Cloud SaaS Team`.
+- For internal Sourcegraph teammates, join us in [#cloud-saas](https://sourcegraph.slack.com/archives/cloud-saas) to ask questions or request help from our team.
+- For cloud users with feature requests, please reach out to our product manager, Ryan, at [ryphil@sourcegraph.com](mailto:ryphil@sourcegraph.com) and include `Cloud Feature Request:` in your subject line.
+
+### Planning, execution, and issue tracking
+
+The Cloud SaaS team is planning work based on our [long-term roadmap](https://docs.google.com/spreadsheets/d/1vTEWyiNQLhJYxo99HkgOhtdNyYfSlyyRNuhEnIySMMc/edit#gid=0) and setting [quarterly goals](#goals-and-roadmap). During the quarter, we are following a flavor of the SCRUM process with biweekly sprints. Our cycle starts every second Tuesday with a retrospective, sprint review, and planning meetings. We set goals for each sprint and focus team efforts during the iteration on achieving these goals rather than closing a number of issues. It’s the outcomes and delivered customer value, not the output, that matters.
```

Comment:
```suggestion
The Cloud SaaS team plans work based on our [long-term roadmap](https://docs.google.com/spreadsheets/d/1vTEWyiNQLhJYxo99HkgOhtdNyYfSlyyRNuhEnIySMMc/edit#gid=0) and setting [quarterly goals](#goals-and-roadmap). During the quarter, we follow a flavor of the SCRUM process with biweekly sprints. Our cycle starts every second Tuesday with a retrospective, sprint review, and planning meetings. We set goals for each sprint and focus team efforts during the iteration on achieving these goals rather than closing a number of issues. It’s the outcomes and delivered customer value, not the output, that matters.
```